cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionDiffCommand.java

```diff
@@ -213,7 +243,33 @@ public void execute() throws CommandException {
     }
   }
 
-  void executeUnsafe() throws CommandException {
+  private void checkMaxStringsBlock(ExtractionDiffPaths extractionDiffPaths)
+      throws MissingExtractionDirectoryExcpetion, CommandException {
+    if (!this.skipMaxStringsBlock
+        && !Strings.isNullOrEmpty(this.pushToBranchName)
+        && !this.pushToBranchName.equals(MASTER_BRANCH_NAME)) {
+      ExtractionDiffStatistics extractionDiffStatistics =
+          this.extractionDiffService.computeExtractionDiffStatistics(extractionDiffPaths);
+      if (extractionDiffStatistics.getAdded() > this.maxStringsBlock) {
+        this.notificationSender.sendMessage(
+            String.format(
+                "The branch: %s has more than %d strings added",
+                this.pushToBranchName, this.maxStringsBlock));
+        throw new CommandException(
+            String.format("There are more than %d strings added", this.maxStringsBlock));
+      }
+      if (extractionDiffStatistics.getRemoved() > this.maxStringsBlock) {
```

Comment:
Can you add a separate parameter for max added/ max deleted so we can configure different thresholds if required?

Reply:
Done